tests/e2e/tests/regression/cli/cli-re-cluster.e2e.ts

```diff
@@ -0,0 +1,111 @@
+    });
+test
+    .meta({ rte: rte.reCloud })
+    .before(async() => {
+        await acceptLicenseTermsAndAddRECloudDatabase(cloudDatabaseConfig);
+    })
+    .after(async() => {
+        //Clear and delete database
+        await browserPage.deleteKeyByName(keyName);
+        await deleteDatabase(cloudDatabaseConfig.databaseName);
+    })('Verify that user can add data via CLI in RE Cloud DB', async t => {
+        keyName = common.generateWord(10);
+        //Open CLI
+        await t.click(cliPage.cliExpandButton);
+        //Add key from CLI
+        await t.typeText(cliPage.cliCommandInput, `SADD ${keyName} "chinese" "japanese" "german"`);
+        await t.pressKey('enter');
+        //Check that the key is added
+        await browserPage.searchByKeyName(keyName);
+        const isKeyIsDisplayedInTheList = await browserPage.isKeyIsDisplayedInTheList(keyName);
+        await t.expect(isKeyIsDisplayedInTheList).ok('The key is added');
+    });
+test
+    .meta({ rte: rte.ossCluster })
+    .before(async() => {
+        await acceptLicenseTermsAndAddOSSClusterDatabase(ossClusterConfig, ossClusterConfig.ossClusterDatabaseName);
+    })
+    .after(async() => {
+        //Clear and delete database
+        await browserPage.deleteKeyByName(keyName);
+        await deleteOSSClusterDatabaseApi(ossClusterConfig);
+    })('Verify that user can add data via CLI in OSS Cluster DB', async t => {
+        keyName = common.generateWord(10);
+        //Open CLI
+        await t.click(cliPage.cliExpandButton);
+        //Add key from CLI
+        await t.typeText(cliPage.cliCommandInput, `SADD ${keyName} "chinese" "japanese" "german"`);
+        await t.pressKey('enter');
+        //Check that the key is added
+        await browserPage.searchByKeyName(keyName);
+        const isKeyIsDisplayedInTheList = await browserPage.isKeyIsDisplayedInTheList(keyName);
+        await t.expect(isKeyIsDisplayedInTheList).ok('The key is added');
+    });
+test
+    .meta({ env: env.web, rte: rte.sentinel })
+    .before(async() => {
+        await acceptLicenseTermsAndAddSentinelDatabaseApi(ossSentinelConfig);
+    })
+    .after(async() => {
+        //Clear and delete database
+        await browserPage.deleteKeyByName(keyName);
+        await deleteSentinelDatabaseApi(ossSentinelConfig);
+    })('Verify that user can add data via CLI in Sentinel Primary Group', async t => {
+        keyName = common.generateWord(10);
+        //Open CLI
+        await t.click(cliPage.cliExpandButton);
+        //Add key from CLI
+        await t.typeText(cliPage.cliCommandInput, `SADD ${keyName} "chinese" "japanese" "german"`);
+        await t.pressKey('enter');
+        //Check that the key is added
+        await browserPage.searchByKeyName(keyName);
+        const isKeyIsDisplayedInTheList = await browserPage.isKeyIsDisplayedInTheList(keyName);
+        await t.expect(isKeyIsDisplayedInTheList).ok('The key is added');
```

Comment:
Let's make a function like you already did in the other test

Reply:
yes, changed, thanks